piranha/src/test/java/com/uber/piranha/XPFlagCleanerTest.java

```diff
@@ -102,7 +189,91 @@ public void positiveControl() throws IOException {
       bcr.doTest();
     } catch (ParseException pe) {
       pe.printStackTrace();
-      fail("Incorrect parameters passed to the checker");
+      assert_().fail("Incorrect parameters passed to the checker");
+    }
+  }
+
+  private BugCheckerRefactoringTestHelper addHelperClasses(BugCheckerRefactoringTestHelper bcr)
+      throws IOException {
+    return bcr.addInputLines(
+            "XPTest.java",
```

Comment:
Can we place them as part of the refactoring files? It is hard to parse them manually and also putting them as part of the refactoring files will ensure consistency and list the respective use cases.